Tools/LyTestTools/tests/integ/sanity_tests.py

```diff
@@ -46,6 +46,7 @@ def test_StartGameLauncher_Sanity(self, project):
         """
         # Kill processes that may interfere with the test
         process_utils.kill_processes_named(names=process_utils.LY_PROCESS_KILL_LIST, ignore_extensions=True)
+        assert False
```

Comment:
This will be removed after testing is finished.

Reply:
When will testing finish? I don't think this change should be merged.